portal/account/account.php

```diff
@@ -31,6 +33,27 @@
     $_SESSION['language_choice'] = (int) $_REQUEST['value'];
     echo 'okay';
     exit();
+} elseif ($action == 'userIsUnique') {
```

Comment:
It looks like I can protect this function, but not allowing use for the 1st comparison above (ie. the registration thing), which will prevent unauthorized bad actors from doing brute force attacks to identify active portal_login_username. Gonna do some testing. @sjpadgett , this is basically the last thing to do here; I should have it completed and ready for commit tomorrow evening; btw, the testing on all these features have been super intuitive and awesome (ie. very nice work!).

Reply:
for me:
this comparison will cover the script call from portal/account/index_reset.php:
isset($_SESSION['pid']) && isset($_SESSION['patient_portal_onsite_two'])

Reply:
for me:
the third comparison above looks good for the call in portal/index.php (just need to ensure the path to this involves auth check; I know it does for when $_SESSION['password_update'] == 2, just need to ensure also does for when it equals 1; going to bed now, but will confirm this tomorrow)

Reply:
ok, just need to test it one more time and then is ready for codebase